ui/admin/tests/acceptance/credential-store/read-test.js

```diff
@@ -54,53 +56,68 @@ module('Acceptance | credential-stores | read', function (hooks) {
     urls.staticCredentialStore = `${urls.credentialStores}/${instances.staticCredentialStore.id}`;
     urls.vaultCredentialStore = `${urls.credentialStores}/${instances.vaultCredentialStore.id}`;
     urls.unknownCredentialStore = `${urls.credentialStores}/foo`;
+    
     authenticateSession({});
+    await visit(urls.projectScope);
```

Comment:
Love the change of adding the initial `visit()` here since all test need it. I would recommend changing this to `await visit(urls.credentialStores);` though. The list of credential stores is where each of these test should start and removes the initial `click()` you have in each test.
```suggestion
    await visit(urls.credentialStores);
```

Reply:
I think this makes sense if you're testing the credential stores as we should be able to start there but i think we have some tests where we test if we have permissions to goto the credential stores page and it makes sense to start on the project page. I think we either have to start on the project page initially or just have the initial navigation handled in each test.

Reply:
I think I am leaning towards removing this from the beforeEach and going back to a visit at the beginning of each test.